src/bert/utils.py

```diff
@@ -12,6 +12,19 @@
 # huggingface implementation of BERT
 import pytorch_pretrained_bert
 
+SEP_ID = 104 # [SEP] for BERT models.
+
+def _get_seg_ids(ids):
+    """ Dynamically build the segment IDs for a concatenated pair of sentences
```

Comment:
Can you give input and output examples in the docstring? (this is pretty opaque otherwise)

Reply:
see below!